agrest-spring-boot-starter/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=io/agrest.spring.starter.AgrestAutoConfiguration
```

Comment:
I am not a Spring expert. So this is more of a question. Is it the correct syntax - `io/agrest.spring.starter....` (the first separator in the class name is a slash, the rest are dots)?

Reply:
Yep, good catch. thanks.

Reply:
Maybe we can create a test for this condition

Reply:
Yep,I'll try. The problem here is that starter works as it is. But that syntax seems confusing for me too.